### PR TITLE
Update github action for Trow

### PR DESCRIPTION
**Pull request by pmengelbert**
Build image from master instead of using pre-built image.

### CODE REVIEW

**Comment by pmengelbert** [comment]
@caniszczyk

File: .github/workflows/trow_1.yml [hunk 1]
```diff
@@ -9,15 +9,30 @@ on:
...
+          path: trow
+          repository: ContainerSolutions/trow
+      - name: Start trow server
+        run: |
+          cd ./trow
+          IP="$(hostname -I | awk '{print $1}')"
+          echo "::set-env name=OCI_ROOT_URL::http://${IP}:8000"
+          TROW_REF="local-trow:v$(date +%Y%m%d%H%M%S)"
+          docker build -f docker/Dockerfile -t "${TROW_REF}" .
+          docker run --rm -itd -p 8000:8000 "${TROW_REF}" --no-tls
```
Comment (amouat) [inline .github/workflows/trow_1.yml:31]: Why the `-it`?

**Comment by amouat** [comment]
I didn't know you could just do `docker run` in an action! Do you know if it uses the same daemon as the current container is running in? (i.e. is it using the host daemon or is there another Docker daemon inside the container)

Be warned that building the image will take a while and you're doing it 3 times. You are welcome to use the `containersol/trow:notls`, which also tracks master. I understand if you'd rather build straight from source though.

Thanks for working on this.

File: .github/workflows/trow_1.yml (hunk 1 above)
Comment (pmengelbert) [reply .github/workflows/trow_1.yml:31]: Old habits die hard -- that's a quick fix though.

**Comment by pmengelbert** [comment]
> I didn't know you could just do `docker run` in an action! Do you know if it uses the same daemon as the current container is running in? (i.e. is it using the host daemon or is there another Docker daemon inside the container)
> 
> Be warned that building the image will take a while and you're doing it 3 times. You are welcome to use the `containersol/trow:notls`, which also tracks master. I understand if you'd rather build straight from source though.
> 
> Thanks for working on this.

No problem -- we're happy to help. Not sure about which daemon is running the container. 

We are only running the job once a day on a schedule, so the build time isn't too important. I think the desire to build and test locally is for transparency more than anything. Thanks for the comments :)

**Comment by pmengelbert** [comment]
@caniszczyk @amouat We should be good to merge here, unless you have anything further to add

**Comment by amouat** [comment]
LGTM!

**Comment by amouat** [comment]
Is there anything else you need from me? I'd love to get this merged if possible :)

**Pull request merged**